### DISCUSSION TITLE
Custom Metadata for Jingle Mode Playlists

### DISCUSSION THREAD

**Discussion by unudelaradio** [AzuraCast/AzuraCast#6523] [Ideas]
Hello!

💡 It would be great if we could set a custom metadata text for every Jingle Mode Playlist. When a jingle plays, the custom text will appear in the player.

💡 Also, for media files, such as commercials, it would be useful an End Date. After the end date of the promotion or commercial campaign, the file will not be loaded anymore in the queue.

Many thanks,
All the best!

**Comment by BusterNeece** [comment]
Hello! If you don't mind, can you submit these via our feature management page over at https://features.azuracast.com/? That'll let our community vote on it and let us prioritize it too.
